File: SpringBoot/demo/src/main/java/com/example/rps/beans/PaperMaster.java
package com.example.rps.beans;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class PaperMaster {
	
	private String type;
	private String quality;
	private int qoh;
	
	public int getQoh() {
		return qoh;
	}
	public void setQoh(int qoh) {
		this.qoh = qoh;
	}
	private float pricePerUnit;
	
	public float getPricePerUnit() {
		return pricePerUnit;
	}
	public void setPricePerUnit(float pricePerUnit) {
		this.pricePerUnit = pricePerUnit;
	}
	public String getType() {
		return type;
	}
	public void setType(String type) {
		this.type = type;
	}
	public String getQuality() {
		return quality;
	}
	public void setQuality(String quality) {
		this.quality = quality;
	}
	
	
}

File: Angular/satva/src/app/papermaster.ts
export class Papermaster {
    private type: string = '';
    public get Type(): string {
        return this.type;
    }
    public set Type(value: string) {
        this.type = value;
    }
	private quality: string = '';
    public get qual(): string {
        return this.quality;
    }
    public set qual(value: string) {
        this.quality = value;
    }
	private qoh: number = 0;
    public get qOH(): number {
        return this.qoh;
    }
    public set qOH(value: number) {
        this.qoh = value;
    }
	private pricePerUnit: number = 0;
    public get priceperUnit(): number {
        return this.pricePerUnit;
    }
    public set priceperUnit(value: number) {
        this.pricePerUnit = value;
    }
}
